terraform/azure/infrastructure/virtual-machine.j2

```diff
@@ -82,6 +82,24 @@ resource "azurerm_virtual_machine" "{{ specification.name }}" {
   depends_on = [azurerm_network_interface_security_group_association.{{ specification.security_group_association_name }}]
   {%- endif %}
 
-  #TODO:
-  # storage_data_disk
 }
+
+{%- if specification.additional_disks is defined %}
+  {%- for disk in specification.additional_disks %}
+resource "azurerm_managed_disk" "{{ specification.name }}-data-disk-{{ loop.index }}" {
+  name                 = "{{ specification.name }}-data-disk-{{ loop.index }}"
```

Comment:
loop.index0 so disks are marked the same as other resources which start from 0.

Reply:
Done.